### ISSUE TITLE
accessibility tag the download button

### ISSUE THREAD

**Issue by casualfan** [rmcrackan/Libation#894]
when I'm navigating your app with my screen reader I can see/read every column except for the download button (the stop light) column next to the book I want to download. the app is amazing. its very easy to use expect for this one issue.

**Comment by rmcrackan** [comment]
Yay, I'm glad you're able to see most of it. My family has quite a bit of disability, albeit not visual, so accessibility is important to me.

* The stoplight column has the header "Liberate".
* For a downloaded book, the green stoplight button's mouse over is "Liberated" and also "PDF downloaded" when that applies.
* For a not downloaded book, the red stoplight button's mouse over is "Book NOT downloaded. Click to complete"

Are you able to see any of this with your screen reader?

**Comment by casualfan** [comment]
No, the software will not read the color unless it's tagged properly. There is a document called accessibility guidelines for creating accessible documents. you should be able to google the guide. I appreciate your app it has been beneficial to me. thank you so much.

**Comment by rmcrackan** [comment]
Sorry, I was unclear. The color isn't important; it's just a visual cue. The mouse-over text I mentioned is the important part; it says whether the book is liberated or if you need to click to download. Is your reader able to see those bits?

**Comment by casualfan** [comment]
No. The screen reader isn't able to read the text because it has to be tagged properly.

**Comment by rmcrackan** [comment]
Ah, I think I see the problem. This was a tricky bit and I'm not sure I got it right. Please try this pre-release I made for you and let me know if it helps -- https://github.com/rmcrackan/Libation/releases/tag/v11.3.9

**Comment by casualfan** [comment]
thanks! ill keep you updated.
